cmd/install/assets/hypershift-operator/hypershift.openshift.io_nodepools.yaml

```diff
@@ -1174,12 +1174,11 @@ spec:
                 type: object
               nodeDrainTimeout:
                 description: 'NodeDrainTimeout is the total amount of time that the
-                  controller will spend on draining a node. The default value is 0,
+                  controller will spend on draining a node. The default value is 0s,
                   meaning that the node can be drained without any time limitations.
-                  NOTE: NodeDrainTimeout is different from `kubectl drain --timeout`
-                  TODO (alberto): Today changing this field will trigger a recreate
-                  rolling update, which kind of defeats the purpose of the change.
-                  In future we plan to propagate this field in-place. https://github.com/kubernetes-sigs/cluster-api/issues/5880'
+                  NOTE: NodeDrainTimeout is different from `kubectl drain --timeout`.
+                  NOTE: Changes to this field won''t trigger a rollout. It''s applied
```

Comment:
small nit: s/''/'/

Reply:
this is autogenerated, not sure why is generated like that though.